features/git-town-ship/current_branch/on_feature_branch/without_open_changes/in_subfolder.feature

```diff
@@ -39,18 +39,18 @@ Feature: git town-ship: shipping the current feature branch from a subfolder
 
 
   Scenario: undo
-    When I run `git town-ship --undo`
+    When I run `gt ship --undo`
     Then it runs the commands
-      | BRANCH  | COMMAND                                        |
-      | <none>  | cd <%= git_root_folder %>                      |
-      | main    | git branch feature <%= sha 'feature commit' %> |
-      |         | git push -u origin feature                     |
-      |         | git revert <%= sha 'feature done' %>           |
-      |         | git push                                       |
-      |         | git checkout feature                           |
-      | feature | git checkout main                              |
-      | main    | git checkout feature                           |
-      | <none>  | cd <%= git_folder "new_folder" %>              |
+      | BRANCH  | COMMAND                                                        |
+      | <none>  | cd <%= git_root_folder %>                                      |
+      | main    | git branch feature <%= sha 'feature commit' %>                 |
+      |         | git push origin <%= sha 'feature commit' %>:refs/heads/feature |
```

Comment:
Should we add the `-u` back to make the remote branch track the local branch? I think that's useful to have.